testsuite/running-tests.md

```diff
@@ -5,55 +5,33 @@ This document will enumerate everything you need to do run the sonobuoy based co
 ## Prerequisites
 
 1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
-1. Set the `KUBECONFIG` environment variable to the path to your kubeconfig file of your cluster.
-2. Install [sonobuoy](https://github.com/vmware-tanzu/sonobuoy#installation) version 0.55.1 or higher. Run `sonobuoy version` to verify it's installed correctly.
+2. Set the `KUBECONFIG` environment variable to the path to your kubeconfig file of your cluster.
 3. Address the [network requirements](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster#meet-network-requirements) on your cluster for the Arc agents to communicate with Azure.
-4. Download and install [git](https://git-scm.com/downloads).
-5. Download and install [az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
-6. Download and install [python 3](https://www.python.org/downloads)
 
-### Additional Prerequisites for Arc enabled Data Services
-
-1. Install [arcdata extension](https://docs.microsoft.com/en-us/azure/azure-arc/data/release-notes).
 
 ## Running the script and publishing the results
 
-1. Clone this repository.
-2. Edit the [`partner-metadata.md`](partner-metadata.md) file and fill in the required details. For reference, please see the [`partner-metadata-sample.md`](partner-metadata-sample.md) file.
-
-### Arc enabled Kubernetes
```

Comment:
We will need to maintain this distinction. Arc enabled data services will continue to leverage their existing scripts to run the indirectly connected mode tests.

Reply:
ack